pkg/vcdsdk/gateway.go

```diff
@@ -1478,19 +1478,17 @@ func (gm *GatewayManager) IsUsingIpSpaces() (bool, error) {
 	}
 
 	edgeGatewayUplinks := edgeGateway.EdgeGateway.EdgeGatewayUplinks
-	if edgeGatewayUplinks != nil {
-		if len(edgeGatewayUplinks) != 1 {
-			return false, fmt.Errorf("found invalid number of uplinks for gateway [%s], expecting 1", edgeGatewayName)
-		}
-	} else {
-		return false, fmt.Errorf("no uplinks were found for gateway [%s], expecting 1", edgeGatewayName)
+	if edgeGatewayUplinks == nil || len(edgeGatewayUplinks) == 0 {
+		return false, fmt.Errorf("no uplinks were found for gateway [%s], expecting atleast 1 uplink", edgeGatewayName)
```

Comment:
The edge can belong to a vdc group too. The VDC that we have access to in this method is the VDC on which the cluster is hosted, it may or may not have explicit ownership of the edge.
I wonder what is the rational behind logging the VDC name.

Reply:
That is a good point. I was asking because the vdc is the object on which we run the query, and in the past we have had various issues of multiple VDCs having edges with the same name.

Also, we currently don't support VDC groups and that is an open bug: https://bugzilla.eng.vmware.com/show_bug.cgi?id=3397729. Could you check with Kiran if this also should be tackled and released as part of this patch? (I think main may already have the fix. @lzichong could confirm about that.)

Reply:
But in either case, this is a nit and not a must-change. The change is LGTM from my side and it can be merged.

Reply:
I will include the other fix in CPI 1.6.1 too.